compose/container.py

```diff
@@ -46,6 +46,24 @@ def create(cls, client, **options):
         response = client.create_container(**options)
         return cls.from_id(client, response['Id'])
 
+    @classmethod
+    def create_exec(cls, client, c_id, command, **options):
+        privileged = options.get("privileged")
+        user = options.get("user")
+        tty = options.get("tty")
+        return client.exec_create(c_id, command, privileged=privileged, tty=tty, stdin=tty,
+                                  user=user)
```

Comment:
I think `c_id` here should come from `self.id`, so it can be removed from the method args as well.